frontend/src/components/landing/carousel/Carousel.tsx

```diff
@@ -13,9 +13,10 @@ type StaticImageData = {
 };
 
 export type CarouselTab = {
+  alt: string;
```

Comment:
I don't think this is actually used? (And I don't see where it would need to be used either :smile: )

```suggestion
```